go/cli/mcap/cmd/version.go

```diff
@@ -3,22 +3,27 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/foxglove/mcap/go/mcap"
 	"github.com/spf13/cobra"
 )
 
 var Version string
+var printLibraryVersion bool
 
 // versionCmd represents the version command
-func versionCmd(version string) *cobra.Command {
-	return &cobra.Command{
-		Use:   "version",
-		Short: "Output version information",
-		Run: func(cmd *cobra.Command, args []string) {
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Output version information",
+	Run: func(cmd *cobra.Command, args []string) {
+		if printLibraryVersion {
+			fmt.Println(mcap.Version)
+		} else {
 			fmt.Println(Version)
-		},
-	}
+		}
```

Comment:
Why do we want the library and CLI versions to be separate concepts at all?

Reply:
They seem separate to me 😅 A breaking change in the mcap library vs. mcap CLI API are pretty different things. To the extent that we have any users of the mcap library directly, they shouldn't care about what happens with the CLI when updating versions.

Reply:
So to summarize the changes, the library version is in the code, and the cli version is still coming from the version variable at build time, right?

Reply:
correct.